.travis.yml

```diff
@@ -145,7 +143,9 @@ script:
 
 after_success:
 - >
-  if [ -n "$TRAVIS_TAG" ]; then
+  if [ "$TRAVIS_PYTHON_VERSION" == "3.5" && "$TRAVIS_OS_NAME" == "osx"]; then
```

Comment:
nit: you can test this only once and add both uploading to pypi and pypi test in 1 else if loop... feel free to ignore this